package.json

```diff
@@ -15,12 +15,16 @@
     "swagger-express-mw": "^0.1.0"
   },
   "devDependencies": {
+    "eslint": "^3.18.0",
+    "npm-run-all": "^4.0.2",
     "should": "^7.1.0",
     "supertest": "^1.0.0"
   },
   "scripts": {
     "postinstall": "npm install -g multi-file-swagger",
     "start": "cd ./api/swagger/src/ && multi-file-swagger ./swagger.yaml > ../swagger.json && cd ../../../ && node app.js",
-    "test": "swagger project test"
+    "test": "npm-run-all test:swagger lint",
+    "test:swagger": "swagger_swagger_fileName=api/swagger/src/swagger.yaml swagger project test",
```

Comment:
`cd ./api/swagger/src/ && multi-file-swagger ./swagger.yaml > ../swagger.json && cd ../../../` will compile the `swagger.yaml` file. The `cd`s are actually necessary as the compiler tool will fail when it isn't being run in the same directory as the `swagger.yaml` file.